pstatus-notifications-workflow-ktor/src/main/kotlin/Application.kt

```diff
@@ -41,7 +53,7 @@ fun Application.module() {
     }
 
     //***ONLY FOR QUICK AND DIRTY TESTING***
-  //  testUploadDigestCount()
+    //  testUploadDigestCount()
```

Comment:
Can this change be removed since commented out anyway?

Reply:
Removed